spec/rubocop/config_loader_spec.rb

```diff
@@ -121,7 +121,7 @@
       end
 
       it 'disables cops by default' do
-        expect(configuration_from_file['Style/Alias']['Enabled']).to eql(false)
+        expect(configuration_from_file['Style/Alias']['Enabled']).to be(false)
```

Comment:
You can do `be_false` or `be_falsey` but they both assert basically "value is false or nil." Writing `be(false)` asserts that the value can only be `false` and nothing else